app/views/giraffe/contributeReact.scala.html

```diff
@@ -0,0 +1,69 @@
+@import com.gu.i18n.CountryGroup
```

Comment:
Rebasing accident. Will fix in morning.

On 7 Sep 2016 18:06, "Patricio Vighi" <notifications@github.com> wrote:

In app/views/giraffe/contributeReact.scala.html
<https://github.com/guardian/contributions-frontend/pull/98#discussion_r77864074>
:

> @@ -0,0 +1,69 @@
> +@import com.gu.i18n.CountryGroup

is there any reason we have this file back? this should now be in
contribute.scala.html

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/guardian/contributions-frontend/pull/98/files/34eeabec7e85d90d5c9399b5d5f623c822af16cc#r77864074>,
or mute the thread
<https://github.com/notifications/unsubscribe-auth/ACi_oEAr8M5vBuFH8NY4qClL-ewZL2zqks5qnu5-gaJpZM4J3H-Z>
.

-- 

------------------------------
This e-mail and all attachments are confidential and may also be 
privileged. If you are not the named recipient, please notify the sender 
and delete the e-mail and all attachments immediately. Do not disclose the 
contents to another person. You may not use the information for any 
purpose, or store, or copy, it in any way.  Guardian News & Media Limited 
is not liable for any computer viruses or other material transmitted with 
or as part of this e-mail. You should employ virus checking software.
 
Guardian News & Media Limited is a member of Guardian Media Group plc. Registered 
Office: PO Box 68164, Kings Place, 90 York Way, London, N1P 2AP.  Registered 
in England Number 908396